modules/mfa/src/Auth/Process/Mfa.php

```diff
@@ -416,15 +418,16 @@ protected static function isHeadedToMfaSetupUrl($state, $mfaSetupUrl)
      *
      * @param int $mfaId The ID of the MFA option used.
      * @param string $employeeId The Employee ID that this MFA option belongs to.
-     * @param string $mfaSubmission The value of the MFA submission.
+     * @param string|array $mfaSubmission The value of the MFA submission.
```

Comment:
Is there a way to single data type the parameter? If it is a string, you can always call with [ $string ] to make it an array.

Reply:
It comes from an external API, so we don't have control of it in this repo. In fact, I think it comes from two different services, which may be why it's a mixed type. And it gets passed to another API, so there's not much point in trying to fix it, except on a system level. Oh, the nastiness of microservices.